ush/python/pygfs/task/aero_analysis.py

```diff
@@ -82,29 +82,18 @@ def initialize(self: Analysis) -> None:
         jedi_fix_list = parse_j2yaml(self.task_config.JEDI_FIX_YAML, self.task_config)
         FileHandler(jedi_fix_list).sync()
 
-        # stage berror files
-        # copy BUMP files, otherwise it will assume ID matrix
-        if self.task_config.get('STATICB_TYPE', 'identity') in ['bump']:
-            FileHandler(self.get_berror_dict(self.task_config)).sync()
-
-        # stage backgrounds
-        FileHandler(self.get_bkg_dict(AttrDict(self.task_config, **self.task_config))).sync()
+        # stage files from COM and create working directories
+        logger.info(f"Staging files prescribed from {self.task_config.AERO_STAGE_VARIATIONAL_TMPL}")
+        aero_var_stage_list = parse_j2yaml(self.task_config.AERO_STAGE_VARIATIONAL_TMPL, self.task_config)
+        FileHandler(aero_var_stage_list).sync()
 
         # generate variational YAML file
         logger.debug(f"Generate variational YAML file: {self.task_config.jedi_yaml}")
         save_as_yaml(self.task_config.jedi_config, self.task_config.jedi_yaml)
         logger.info(f"Wrote variational YAML to: {self.task_config.jedi_yaml}")
 
-        # need output dir for diags and anl
-        logger.debug("Create empty output [anl, diags] directories to receive output from executable")
-        newdirs = [
-            os.path.join(self.task_config['DATA'], 'anl'),
-            os.path.join(self.task_config['DATA'], 'diags'),
-        ]
-        FileHandler({'mkdir': newdirs}).sync()
-
     @logit(logger)
-    def execute(self: Analysis) -> None:
+    def variational(self: Analysis) -> None:
```

Comment:
Best verb I could come up with here, open to others.
```suggestion
    def var_assimilate(self: Analysis) -> None:
```